vulnerabilities/importers/ubuntu_usn.py

```diff
@@ -74,7 +75,7 @@ def to_advisories(usn_db):
                 # The db sometimes contains entries like
                 # {'cves': ['python-pgsql vulnerabilities', 'CVE-2006-2313', 'CVE-2006-2314']}
                 # This `if` filters entries like 'python-pgsql vulnerabilities'
-                if not cve.startswith("CVE-"):
+                if not is_cve(cve):
                     continue
```

Comment:
skipping entirely would be bad idea as we'll be losing other data. 

Instead I suggest 

```
  if not is_cve(cve):
                    cve = ""
```

Reply:
Yes, It's a good idea as empty CVE will automatically takes the VULCOID-X-X-X.